webapp/display/changes/build_text.js

```diff
@@ -2,18 +2,17 @@ import React, { PropTypes } from 'react';
 import moment from 'moment';
 import { OverlayTrigger, Tooltip } from 'react_bootstrap';
 
-import { LiveTime, display_duration } from 'es6!display/time';
-import { get_runnable_condition,
-         get_runnables_summary_condition,
-         COND_PASSED,
+import { COND_PASSED,
          COND_FAILED,
-         COND_FAILED_ABORTED,
          COND_FAILED_INFRA,
+         COND_FAILED_ABORTED,
```

Comment:
Why is the order of this one getting shifted around? Why doesn't the import script put things in alphabetical order?